gemsfx-demo/src/main/resources/com/dlsc/gemsfx/demo/svg-image-view-app.css

```diff
@@ -0,0 +1,26 @@
+.control-box {
+    -fx-alignment: top-center;
+    -fx-spacing: 10;
+    -fx-padding: 10px;
+    -fx-min-width: 185px;
+    -fx-max-width: 185px;
+}
+
+.control-box > .check-box,
+.control-box > .spinner,
+.control-box > .button{
+    -fx-pref-width: 165px;
+}
+
+.svg-image-view {
+    /*-fx-svg-url: url("microphone.svg");*/
```

Comment:
If the svgUrl property is set using CSS, the setSVGUrl(url) in the code will be invalidated, because the final effect will always be affected by the CSS setting.